pkg/common/message/msghandler.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
This doesnt seem to be a common package.

message also does not convey the meaning of this code.

Reply:
still figuring out good place for it, fixing it in next patch in this PR.

Reply:
moved to pkg/dicomm/msghandler and renamed it to Registrar.

 `msghandler.NewRegistrar(), msghandler.Registrar`